homeworks/yehor.honcharenko_NOcode-developer/GamesProject/About/index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Game Project</title>
+        <link rel="stylesheet" href="style.css">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    </head>
+    <body>
+        <section class="wrapper">
+            <header>
+                <nav class="nav__container">
```

Comment:
If you like to use BEM, you should start from the `block`. Please read tru my comments from this [pr](https://github.com/MastersAcademy/fe-course-basic-2023/pull/136#discussion_r1367945552)

Reply:
fixed / done ✅